packages/angular/src/generators/library/files/base/tsconfig.json__tpl__

```diff
@@ -0,0 +1,17 @@
+{
+  "compilerOptions": {
+    "target": "es2022",
+    "useDefineForClassFields": false,
+  },
+  "files": [],
+  "include": [],
+  "references": [
+    {
+      "path": "./tsconfig.lib.json"
+    },<% if(unitTesting) { %>
+    {
+      "path": "./tsconfig.spec.json"
+    }<% } %>
```

Comment:
I just realized we shouldn't handle any test-related config & files (except spec files) in the `library` generator. We invoke the `jestProject` generator and that one sets up any file and config needed for a project. It handles this change and others we're doing in this PR.

The same applies to what we already did in the `application` generator. We need to amend that.

Reply:
I created a PR for the application generator to address this https://github.com/nrwl/nx/pull/15731.